vendor/github.com/ethereum/go-ethereum/common/bytes.go

```diff
@@ -36,6 +36,15 @@ func FromHex(s string) []byte {
 	return Hex2Bytes(s)
 }
 
+// ToHex returns the hex string representation of bytes with 0x prefix.
+func ToHex(bytes []byte) string {
```

Comment:
Changing anything in `vendor/*` is a bad idea, it will be vanished with `make vendor`

Reply:
yes, pipeline already pointed me on my mistake :) 
Fixed